editor/editor_dock_manager.cpp

```diff
@@ -327,6 +362,44 @@ void EditorDockManager::_dock_make_selected_float() {
 	_edit_current();
 }
 
+void EditorDockManager::filesystem_dock_show_placement_popup(const Rect2i &p_position) {
```

Comment:
If the system is meant more generic, this method shouldn't specifically target FileSystemDock.